### PR TITLE
Alex The Creator: added translations fr

### PR DESCRIPTION
**Pull request by VanSlanzar**
**A9S:** just added translations fr
**A10S:** added translations fr and remove `\\` useless before `/charge` which creates an error
a10s error

**A11S+txt:** added translations fr and change name E.D.D. which creates an error on`replacesync`
a11s e d d
**after change txt.** I tested triggers and timeline in the instance. Everything is work well.
a11s edd txt
a11ss edd js

**A12S:** added translation fr and remove space on `gainsEffect` which creates an error and it's incorrect for change like `effectNames`. Everything is work well.
A12s gains eefect

Thank you @quisquous for help me to resolve a10s error and answer my questions regularly. Thanks to that, I was able to understand and correct the other errors.

@mooondark, if you want check my translations, a second opinion is always useful.

### CODE REVIEW

File: ui/raidboss/data/03-hw/raid/a11s.js [hunk 1]
```diff
@@ -219,10 +225,10 @@
           return {
             en: 'Knockback Cleave; Face Outside',
             de: 'Rückstoß Cleave; nach Außen schauen',
+            fr: 'Poussée Cleave; Regardez à l\'extérieur',
```
Comment (mooondark) [inline ui/raidboss/data/03-hw/raid/a11s.js:228]: _Regardez vers l\'extérieur_ is better, don't you think ?

File: ui/raidboss/data/03-hw/raid/a12s.js [hunk 2]
```diff
@@ -166,35 +169,36 @@
...
-      regexFr: Regexes.gainsEffect({ effect: 'Jugement : diffamation' }),
+      regexFr: Regexes.gainsEffect({ effect: 'Jugement: diffamation' }),
       regexJa: Regexes.gainsEffect({ effect: '確定判決：名誉罰' }),
       regexCn: Regexes.gainsEffect({ effect: '判决确定：名誉罪' }),
       regexKo: Regexes.gainsEffect({ effect: '확정 판결: 명예형' }),
       condition: Conditions.targetIsYou(),
       alarmText: {
         en: 'Defamation',
         de: 'Ehrenstrafe',
+        fr: 'diffamation',
```
Comment (mooondark) [inline ui/raidboss/data/03-hw/raid/a12s.js:198]: Diffamation (missing capital)

File: ui/raidboss/data/03-hw/raid/a12s.js [hunk 3]
```diff
@@ -308,19 +313,22 @@
...
+        'The General\'s Might': 'Pouvoir d\'Arrhabée',
+        'The General\'s Time': 'Temps d\'Arrhabée',
+        'The General\'s Wing': 'Aile d\'Arrhabée',
       },
       'replaceText': {
-        '(?<!Radiant )Sacrament': 'Sacrement',
-        'Almost Holy': 'Quasi-Miracle',
-        'Arrhidaeus\'s Lanner': 'messager d\'Arrhabée',
+        '(?<! )Sacrament': 'Sacrement',
+        '--timestop--': 'arrêt du temps',
```
Comment (mooondark) [inline ui/raidboss/data/03-hw/raid/a12s.js:326]: missing --

File: ui/raidboss/data/03-hw/raid/a9s.js [hunk 4]
```diff
@@ -128,6 +135,7 @@
         return {
           en: 'Scrapline on YOU',
           de: 'Schrottlinie auf DIR',
+          fr: 'Corde à feraille sur VOUS',
```
Comment (mooondark) [inline ui/raidboss/data/03-hw/raid/a9s.js:138]: fer**r**aille

File: ui/raidboss/data/03-hw/raid/a11s.js (hunk 1 above)
Comment (VanSlanzar) [reply ui/raidboss/data/03-hw/raid/a11s.js:228]: J'ai honte, ça n'a aucun sens. Bon il était très tard.

File: ui/raidboss/data/03-hw/raid/a12s.js (hunk 2 above)
Comment (VanSlanzar) [reply ui/raidboss/data/03-hw/raid/a12s.js]: Merci

File: ui/raidboss/data/03-hw/raid/a12s.js (hunk 3 above)
Comment (VanSlanzar) [reply ui/raidboss/data/03-hw/raid/a12s.js]: Merci

File: ui/raidboss/data/03-hw/raid/a9s.js (hunk 4 above)
Comment (VanSlanzar) [reply ui/raidboss/data/03-hw/raid/a9s.js]: Merci.

File: ui/raidboss/data/03-hw/raid/a11s.txt [hunk 5]
```diff
@@ -26,7 +26,7 @@ hideall "--sync--"
 40.6 "Spin Crusher" sync /:Cruise Chaser:1A85:/
 
 # skippable
-45.8 "E.D.D. Add" sync / 03:........:Added new combatant E\.D\.D\./ window 50,5
+45.8 "E.D.D. Add" sync / 03:........:Added new combatant E.D.D./ window 50,5
```
Comment (quisquous) [inline ui/raidboss/data/03-hw/raid/a11s.txt:29]: Can you leave the `E\.D\.D\.` and change your replacement sync to `E\\\\.D\\\\.D\\\\.`? (sorry)

Comment (VanSlanzar) [reply ui/raidboss/data/03-hw/raid/a11s.txt]: It's ok.

**Comment by VanSlanzar** [comment]
Everything has been modified according to requests, it's good for me.

**Pull request merged**